extensions/cornerstone/src/components/WindowLevelActionMenu/VolumeLighting.tsx

```diff
@@ -109,7 +117,11 @@ export function VolumeLighting({
             min={0}
             type="range"
             step={0.1}
-            style={{ background: calculateBackground(specular) }}
+            style={{
+              background: calculateBackground(specular),
+              '--thumb-inner-color': '#5acce6',
+              '--thumb-outer-color': '#090c29',
+            }}
```

Comment:
all of these inputs should be our inputRange component

Reply:
The input range component doesn't work for all of them, for shift I have to block calculations when the slider is being dragged, I can modify the input range to support some of this then use it if you would like